projects/storefrontapp-e2e-cypress/cypress/helpers/checkout-as-persistent-user.ts

```diff
@@ -76,7 +76,7 @@ export function selectShippingAddress() {
     .first()
     .find('.cx-summary-amount')
     .should('not.be.empty');
-  cy.get('.cx-card-title').contains(' Default Shipping Address ');
+  cy.get('.cx-card-body__title').contains('Default Shipping Address');
```

Comment:
same as above, better use `.should(...)` instead of `.contains(...)`.

Reply:
Done